v2.2/managed-frequently-asked-questions.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Frequently Asked Questions
+summary:
+toc: true
+build_for: [managed]
+---
+
+## Cluster basics
+
+### Is my cluster secure?
+
+Yes. We create individual sub-accounts and VPCs for each cluster within the cloud provider. These VPCs are firewalled from each other and any other outside connection, unless whitelisted for SQL and Web UI ports. The whitelist is an additional layer of protection for your cluster. Connections will only be accepted if they come from a whitelisted IP address, which protects against both compromised passwords and any potential bugs in the server.
+
+We use separate certificate authorities for each cluster, and all connections to the cluster over the internet use TLS 1.2.
+
+### Is my cluster isolated? Does it share resources with any other clusters?
+
+Managed CockroachDB is a single-tenant offering and resources are not shared between clusters.
```

Comment:
Is the term "single-tenant" an industry standard? If not, we should probably define it somewhere. 
Another question that stems from my lack of knowledge: single-tenant = one cluster per customer, right? Can a customer have multiple clusters? And in that case, can the resources be shared within clusters belonging to the same customer?